include/crm/msg_xml.h

```diff
@@ -346,6 +346,8 @@ extern "C" {
 #  define XML_CONFIG_ATTR_FORCE_QUIT	"shutdown-escalation"
 #  define XML_CONFIG_ATTR_RECHECK	"cluster-recheck-interval"
 #  define XML_CONFIG_ATTR_FENCE_REACTION	"fence-reaction"
+#  define XML_CONFIG_ATTR_SHUTDOWN_LOCK     "shutdown-lock"
+#  define XML_CONFIG_ATTR_SHUTDOWN_LOCK_EXP "shutdown-lock-expiration"
```

Comment:
That is rather a lifetime than a point in the future when the lock should expire - right.
Naming triggers rather the latter with me.

Reply:
Renamed to shutdown-lock-limit